service/models.py

```diff
@@ -18,10 +18,142 @@ def init_db(app):
     """ Initializes the SQLAlchemy app """
     Customer.init_db(app)
 
-
 class DataValidationError(Exception):
     """ Used for an data validation errors when deserializing """
 
+class Address(db.Model):
+    """
+    Class that represents a Address
+    """
+
+    ################
+    # Address Schema
+    ################
+
+    logger = logging.getLogger('flask.app')
+    app = None
+
+    # Table Schema
+    address_id = db.Column(db.Integer, primary_key=True)
```

Comment:
"id" should be fine right, to keep things uniform in each model. as in, in customer table we have id.

Reply:
Having "address_id" here instead of having "id" to remove the confusion with "id" present in the customer which denotes customer id, hope that clears it.